app/assets/images/payment_icons/dpo-tigopesa.svg

```diff
@@ -0,0 +1 @@
QAAAAAAJCYMAgAAAAAASEwYBAAAAAAAkJgwCAAAAAAAIDFhEAAAAAAAQGLCIAAAAAAAgMSEQQAA AAAAAIkJgwAAAAAAABITBgEAAAAAACQmDAIAAAAAAEhMGAQAAAAAAJCYMAgAAAAAACAxYRAAAAAA AEBiwiAAAAAAAIDEhEEAAAAAAACJCYMAAAAAAAASEwYBAAAAAAAkJgwCAAAAAABITBgEAAAAAACQ mDAIAAAAAAAgMWEQAAAAAABAYsIgAAAAAACAxIRBAAAAAAAAiQmDAAAAAAAAEhMGAQAAAAAAJCYM AgAAAAAASEwYBAAAAAAAkJgwCAAAAAAAIDFhEAAAAAAAQGLCIAAAAAAAgMSEQQAAAAAAAIkJgwAA AAAAABITBgEAAAAAACQmDAIAAAAAAEhMGAQAAAAAAJCYMAgAAAAAACAxYRAAAAAAAEBiwiAAAAAA AIDEhEEAAAAAAACJCYMAAAAAAAASEwYBAAAAAAAkJgwCAAAAAABITBgEAAAAAACQmDAIAAAAAAAg MWEQAAAAAABAYsIgAAAAAACAxIRBAAAAAAAAiQmDAAAAAAAAEhMGAQAAAAAAJCYMAgAAAAAASEwY BAAAAAAAkJgwCAAAAAAAIDFhEAAAAAAAQGLCIAAAAAAAgMSEQQAAAAAAAIkJgwAAAAAAABITBgEA AAAAACQmDAIAAAAAAEhMGAQAAAAAAJCYMAgAAAAAACAxYRAAAAAAAEBiwiAAAAAAAIDEhEEAAAAA AACJCYMAAAAAAAASEwYBAAAAAAAkJgwCAAAAAABITBgEAAAAAACQmDAIAAAAAAAgMWEQAAAAAABA YsIgAAAAAACAxIRBAAAAAAAAiQmDAAAAAAAAEhMGAQAAAAAAJCYMAgAAAAAASEwYBAAAAAAAkJgw CAAAAAAAIDFhEAAAAAAAQGLCIAAAAAAAgMSEQQAAAAAAAIkJgwAAAAAAABITBgEAAAAAACQmDAIA AAAAAEhMGAQAAAAAAJCYMAgAAAAAACAxYRAAAAAAAEBiwiAAAAAAAIDEhEEAAAAAAACJCYMAAAAA AAASEwYBAAAAAAAkJgwCAAAAAABITBgEAAAAAACQmDAIAAAAAAAgMWEQAAAAAABAYsIgAAAAAACA xIRBAAAAAAAAiQmDAAAAAAAAEhMGAQAAAAAAJCYMAgAAAAAASEwYBAAAAAAAkJgwCAAAAAAAIDFh EAAAAAAAQGLCIAAAAAAAgMSEQQAAAAAAAIkJgwAAAAAAABITBgEAAAAAACQmDAIAAAAAAEhMGAQA AAAAAJCYMAgAAAAAACAxYRAAAAAAAEBiwiAAAAAAAIDEhEEAAAAAAACJCYMAAAAAAAASEwYBAAAA AAAkJgwCAAAAAABITBgEAAAAAACQ1L/+8x9/EgYBAAAAAAAkJgwCAAAAAABITBgEAAAAAACQmDAI AAAAAAAgMWEQAAAAAABAYsIgAAAAAACAxIRBAAAAAAAAiQmDAAAAAAAAEhMGAQAAAAAAJCYMAgAA AAAASEwYBAAAAAAAkJgwCAAAAAAAIDFhEAAAAAAAQGLCIAAAAAAAgMSEQQAAAAAAAIkJgwAAAAAA ABITBgEAAAAAACQmDAIAAAAAAEhMGAQAAAAAAJCYMAgAAAAAACChf/3nP/70+VY/T/8HAAAAAACA HOb5j51BAAAAAAAAiQmDAAAAAAAAEhMGAQAAAAAAJCYMAgAAAAAASEwYBAAAAAAAkNgvYdC//vMf f9LJAAAAAAAA+fx1Z5BACAAAAAAAIB+PiQMAAAAAAEhMGAQAAMD/b+8ObhsEggCKguQrpdGNy6Cb lJYCJsrBEjF2Ygwsu5P3jsiyAR+/ZhYAAEhMDAIAAAAAAEhMDAIAAAAAAEhMDAIAAAAAAEhMDAIA AAAAAEjk8+Paz59GDAIAAAAAAEhMDAIAAAAAAEhMDAIAAAAAAEjsRwy63yEHAAAAAABAOx61HpNB AAAAAAAAifURsXi6YZyWFwEAAAAAAKjSo4mgm4cx6EYUAgAAAAAAqNNvAWjusrgyM/8SYQgAAAAA AKA9zgwCAAAAAABozKtTQZ0YBAAAAAAAkJsYBAAAAAAAkJgYBAAAAAAAkJgYBAAAAAAAkNjFnwsA AAAAAFCvz49rv+Xm+ohYXHxmGKfXPwwAAAAAAMDbtkagG2viAAAAAAAAEhODAAAAAAAAElsVg/Ya RwIAAAAAAKAMk0EAAAAAAACJrY5BpoMAAAAAAADa0UfEppsdxmnbFwAAAAAAAPDDnsM5m2NQJwgB AAAAAAC87eitbLucGWR1HAAAAAAAQJ12iUEAAAAAAACsV2LgRgwCAAAAAABITAwCAAAAAABITAwC AAAAAABIbLcY9L3TrsReOwAAAAAAAF532ftdzYPQME6x+AAAAAAAAADFHLomzqQQAAAAAADAuQ4/ M0gQAgAAAAAAWCrVUA6PQQAAAAAAAJxHDAIAAAAAAEhMDAIAAAAAACis5DE7fUQsLh5hGKcyPwQA AAAAAFCpkhHoplgMekQgAgAAAAAAWndG4Fnj1DVxtb8cAAAAAACA1jkzCAAAAAAAIDExCAAAAAAA IDExCAAAAAAAILHTY5BzgwAAAAAAAI5TxWSQIAQAAAAAAHCMPiKqe7XDONV3UwAAAAAAAHdaGHip Mga9QjACAAAAAABKanXTWRVr4gAAAAAAADiGGAQAAAAAAJCYGAQAAAAAAPCHVlfEdWIQAAAAAABA bs3GoJYLHAAAAAAA0I7Wm0QfEYuLLRnGqe0HAAAAAAAAqpRlMKX5GHRPHAIAAAAAAH7z37aPpTsz yPo4AAAAAADgmf/YEdJNBq1higgAAAAAAHIwLPJE13VfdK3IMLrvUY0AAAAASUVORK5CYII=" transform="translate(2.093 1.031) scale(.0202)" overflow="visible"/></g></svg>
```

Comment:
- The `<clipPath id="pi-dpo-tigopesa-SVGID_2_">` and the `<defs>` don't seem to be used. Can we remove it?
- I think we can remove the `<g clip-path="url(#SVGID_2_)">` and keep the `<image>` only. What are your thoughts?